main.cpp

```diff
@@ -1,6 +1,7 @@
 #include "main.h"
 #include "MainWindow.h"
 
+#include "Editors/PathEditor.h"

```

Comment:
Where would we go with all this initialization? I wanted it to be in the editor classes. We would need to make a more refined API for custom editors too. What if we want to do dynamic translations at runtime later.